### DISCUSSION TITLE
AppleDict Bin Tests

### DISCUSSION THREAD

**Discussion by ilius** [ilius/pyglossary-test#4] [Q&A] [answered]
@soshial What are these numbers after `c` and `t`?
Version numbers of Apple Dictionary?

```
006-en-oxfjord_v10.5_c0_t0.dictionary
006-en-oxfjord_v10.5_c1_t0.dictionary
006-en-oxfjord_v10.6_c0_t0.dictionary
006-en-oxfjord_v10.6_c0_t1.dictionary
006-en-oxfjord_v10.6_c0_t2.dictionary
006-en-oxfjord_v10.6_c0_t3.dictionary
006-en-oxfjord_v10.6_c1_t0.dictionary
006-en-oxfjord_v10.6_c1_t1.dictionary
006-en-oxfjord_v10.6_c1_t2.dictionary
006-en-oxfjord_v10.6_c1_t3.dictionary
006-en-oxfjord_v10.6_c2_t0.dictionary
006-en-oxfjord_v10.6_c2_t1.dictionary
006-en-oxfjord_v10.6_c2_t2.dictionary
006-en-oxfjord_v10.6_c2_t3.dictionary
006-en-oxfjord_v10.11_c2_t3.dictionary
```

**Comment by soshial** [comment, marked as answer]
I used Apple's kit to create binary dictionaries from source XMLs ([standalone version here](https://github.com/SebastianSzturo/Dictionary-Development-Kit) that doesn't need macOS). This kit has several parameters that are identical to dict names:
`compile_dict -v "10.6" -c 1 -t 2`

`v` is format version (depends on macOS version)
`c` is compression
`t` is trie setting
